pkg/volume/flexvolume/plugin.go

```diff
@@ -42,9 +42,30 @@ type flexVolumePlugin struct {
 	unsupportedCommands []string
 }
 
-var _ volume.AttachableVolumePlugin = &flexVolumePlugin{}
+type flexVolumeAttachablePlugin struct {
+	flexVolumePlugin
+}
+
+var _ volume.AttachableVolumePlugin = &flexVolumeAttachablePlugin{}
 var _ volume.PersistentVolumePlugin = &flexVolumePlugin{}
 
+func (plugin *flexVolumeAttachablePlugin) isAttachable() (bool, error) {
+	call := plugin.NewDriverCall(initCmd)
+	res, err := call.Run()
+	if err != nil {
+		return false, err
+	}
+
+	// By default all plugins are attachable, unless they report otherwise.
```

Comment:
To clarify it would break existing post-1.6 drivers right? Pre-1.6 drivers would continue to work as is if the default was false, right?